08_voting/index.tex

```diff
@@ -1 +1,101 @@
+
+\textbf{Changes of rules:} The President has the power to select a rule from the \emph{rule proposal list} to be voted by the Voters. The Speaker will host the voting for this rule and will be responsible for counting the votes in order to declare the result. The votes from the Voters can only be of the following types: \textbf{Aye} (namely agree with the rule change), \textbf{Nay} (namely disagree with the rule change) or \textbf{Abstain}.
+
+
+\section{Voting Methods}
+\label{sec:VotingMethods}
+
+\begin{table}[H]
+\caption{Available voting methods for each voting scenario}\label{table:votingmethod}
+\begin{center}
+\begin{tabular}{ |p{3cm}||p{3cm}|p{3cm}|  }
+ \hline
+ Voting Methods   & Elections of Roles & Changes of Rules   \\
+ \hline
+ Plurality   &     & \checkmark     \\
+ \hline
+ Borda Count &  \checkmark   &      \\
+ \hline
+ Runoff      &  \checkmark    &        \\
+ \hline
+ Instant Runoff    & \checkmark  &     \\
+ \hline
+ Approval  &  \checkmark    &    \\
+ 
+ \hline
+\end{tabular}    
+\end{center}
+\end{table}
+
+Table \ref{table:votingmethod} indicates the available voting method choices for each voting scenario considered in the coursework.
+
+\textbf{Plurality} will select the relative majority choices as the collective decision. In the IIGO session, the Plurality voting method will be used to decide whether a change of rule goes into effect. When the number of valid ballots (i.e. abstention excluded) is odd, the Plurality method can determine the winner without tied results. Otherwise, the solution to tied results is regimented with the assumption that the proposer of the voting in such events will vote for "agree". Thus, the proposal will be approved if the number of "agree" votes is more than the number of "disagree" votes. 
+
+\textbf{Borda Count} method will allow Voters to rank their preference instead of choosing \emph{only} one option. The order of choices in the preference list determines the Borda points regarding each option. Adding such Borda points from all votes produces the Borda score for each candidate. Thus, the candidate with the highest Borda score will be the winner.
+
+\textbf{Runoff} (also known as the two-round system) is a voting method which counts the ballots for two rounds. Voter islands will rank their preference of candidates and submit the preference lists as their ballots. In the first round, candidates with the top-most and second-most first-placed votes are chosen among all candidates. If one of them obtains a majority, the Runoff voting can conclude such candidate as the winner. Otherwise, the second round will proceed where the ballots will be recounted by eliminating the candidates other than the top-two. Subsequently, the candidate with the most votes between the top-two candidates will be the winner.
```

Comment:
suggested re-writing:

If one of them obtains a majority, **the Runoff method** can conclude **this candidate to be the winner**. Otherwise, the second round will proceed where the ballots will be recounted by eliminating the candidates other than **the top two choices**. Subsequently, the candidate with the most votes between the **top two** candidates will be the winner.

Reply:
changed